technical/validations.md

```diff
@@ -7,20 +7,35 @@ these validations will be duplicated in the front-end code for an optimal user e
 ## Entity-level validations
 
 ### Playground
-- The name of the playground is between 3 en 40 characters
-- Two playgrounds cannot be located within .. metres of each other
+- The name of the playground is between 3 en 40 characters ( no whitespaces in the beginning and trailing of the playground name )
+- Two playgrounds cannot be located within .. metres of each other ( Are there any API/Service provided by Netherlands to verify entered/chosen address )
 - The maximum number of playgrounds that can exist within the system is ..
+- At least 4(TBD) decimal precisions required for Longitude and Latitude of the Playground location, in order to identify how close two Playgrounds are located
```

Comment:
I don't think this is a validation. It is an requirement to the front-end, but there is no need to fail on a request that has less decimal places.

Reply:
If all the requests come from front-end, then there is no need to validate.
But, If we are importing playgrounds by calling API or importing any files having playgrounds list, then how are we going to validate the incoming playgrounds data?